### PR TITLE
[FIX] auth_totp: Fix wrong secret key in creation wizard

### LINKED ISSUE #140: auth_totp wrong secret code in wizard
Tested on v11 and v12 runbot, unable to test on v10/v9

When opening the wizard to add a new MFA device the secret code (and QR code) displayed is not the same as the secret code used to validate.

After entering any Confirmation Code and clicking Create, the secret code and QR code reloads and shows the correct code. If this is not noticed by user they continue entering the 6 digit code from the first secret and continue receiving validation errors.

Workaround: Enter any 6 digit code and click validate, then add the displayed secret code / QR to authenticator

### PR DESCRIPTION
**Pull request by p-tombez**
Read-only fields are not passed to the `write` method, as the `secret_key` field is required, the ORM call the default get again, overwriting the values displayed on the form.

Fixes #140

### CODE REVIEW

File: auth_totp/wizards/res_users_authenticator_create.xml [hunk 1]
```diff
@@ -20,7 +20,8 @@
                     <group name="data">
                         <field name="name"/>
                         <field name="user_id"/>
-                        <field name="secret_key" readonly="1"/>
+                        <field name="secret_key" invisible="1" />
+                        <field name="secret_key_display" />
```
Comment (Tonow-c2c) [inline auth_totp/wizards/res_users_authenticator_create.xml:24]:
```suggestion
                        <field name="secret_key_display" readonly="1" />
```

Comment (p-tombez) [reply auth_totp/wizards/res_users_authenticator_create.xml:24]: Already `readonly=True` in the model

**Review by Tonow-c2c — APPROVED** [review]

File: auth_totp/wizards/res_users_authenticator_create.xml (hunk 1 above)
Comment (Tonow-c2c) [reply auth_totp/wizards/res_users_authenticator_create.xml:24]: yes thanks

**Review by grindtildeath — APPROVED** [review]

File: auth_totp/wizards/res_users_authenticator_create.xml [hunk 2]
```diff
@@ -20,7 +20,8 @@
                     <group name="data">
                         <field name="name"/>
                         <field name="user_id"/>
-                        <field name="secret_key" readonly="1"/>
```
Comment (gkliska) [inline auth_totp/wizards/res_users_authenticator_create.xml]: <field name="secret_key" readonly="1" force_save="1"/>

**Review by i-vyshnevska — APPROVED** [review]

**Review by leemannd — APPROVED** [review]

**Review by guewen — APPROVED** [review]

**Comment by guewen** [comment]
Oh runbot fails because of this:

```

2021-03-03 15:55:31,347 172 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (secret_key_display, secret_key) of res.users.authenticator.create() have the same label: Secret Code.
```

**Comment by grindtildeath** [comment]
@p-tombez @gkliska Is this PR still needed after https://github.com/OCA/server-auth/pull/199 was merged? AFAIU it seems to answer the same issue

**Pull request merged**